### PR TITLE
Add logging functionality to understand Xmin wraparound

### PR DESCRIPTION
**Pull request by akashkulk**
- Add logging for DB version (all JDBC sources)
- Add logging for Xmin values

The xmin values will inform us whether 
1. wraparound has occurred
2. Current xmin value associated with database

Tested locally for all version Postgres 9+

### CODE REVIEW

File: airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java [hunk 1]
```diff
@@ -426,6 +426,20 @@ public JdbcDatabase createDatabase(final JsonNode sourceConfig) throws SQLExcept
...
+  /**
+   * {@inheritDoc}
+   * @param database database instance
+   * @param catalog schema of the incoming messages.
+   * @throws SQLException
+   */
+  @Override
+  protected void logPreSyncDebugData(final JdbcDatabase database, final ConfiguredAirbyteCatalog catalog)
+      throws SQLException {
+    LOGGER.info("Data source product recognized as {}:{}",
```
Comment (rodireich) [inline airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java:438]: `.info` or `.debug`?

**Review by rodireich — APPROVED** [review]

File: airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java (hunk 1 above)
Comment (akashkulk) [reply airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java:438]: This one should definitely be info. There will only be one log per sync, and we want it to be visible.

**Comment by akashkulk** [comment]
/publish connector=connectors/source-postgres run-tests=false

**Comment by akashkulk** [comment]
/publish connector=connectors/source-postgres-strict-encrypt run-tests=false

**Comment by akashkulk** [comment]
/publish connector=connectors/source-mysql-strict-encrypt run-tests=false

**Comment by akashkulk** [comment]
/publish connector=connectors/source-mssql-strict-encrypt run-tests=false

**Comment by akashkulk** [comment]
/publish connector=connectors/source-alloydb-strict-encrypt run-tests=false

**Pull request merged**